content/job-tmpfs.md

```diff
@@ -0,0 +1,43 @@
+Job configuration often contains credentials. In some environments we do not want these values to be written to a physical disk and only ever want them to be resident in memory. The director has been able to avoid writing credentials to disk for some time now but they would be written to disk on the deployment VMs.
+
+The job directory on `tmpfs` feature makes sure that the agent does not write these values to disk and instead keeps them on an in-memory `tmpfs`.
+
+## Director Disk
+
+The director can be configured to load credentials from a secret store, perform all job template rendering in-memory, and then send the rendered templates over the message bus directly to the agent. For a complete solution, the `enable_nats_delivered_templates` feature should be enabled at the same time as the `tmpfs` agent feature.
+
+
+```
+- name: bosh
+  ...
+  properties:
+    ...
+    director:
+      ...
+      enable_nats_delivered_templates: true
```

Comment:
Is this a new property or one that's been in the director for a while? 

I'm assuming this manifest is actually part of the director manifest? not the deployment manifest like below?

Reply:
Yep, it's been there for a while. It's a director property.